src/content/docs/en/guides/deploy/space.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Deploy your Astro Site to Space
+description: How to build an Astro site in Space.
+type: deploy
+i18nReady: true
+---
+import InstallGuideTabGroup from '~/components/TabGroup/InstallGuideTabGroup.astro';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[Deta Space](https://deta.space/developers) is a personal computer that lives in the cloud — a ‘personal cloud’. You can build and run apps for yourself in your own "Space". You can publish the apps you've built, and they'll run for people all around the world. 
+
+You can also install apps published by other devs. You'll get a personal copy of the app in your own cloud, running on your infrastructure. All the data that all these apps save to your Space is for you: to explore, to extend, and to export, if you like.
+
+This guide includes step-by-step instructions for building sites in Space. You can build any type of Astro site in Space: either a static site or a server-rendered site.
+
+## Prerequisite
+
+To push an Astro site to Space, make sure you first:
+
+- Create a [Space account](https://deta.space)
+- Install the [Space CLI](https://deta.space/docs/en/basics/cli) and log in
+
+No matter the type of Astro site you are building, create a Space project **inside** the directory of your Astro project. Run the CLI and follow the instructions on the screen.    
+
+```bash
+space new
+```
+
+:::note
+The Space CLI will try to detect the right configuration for your app. Accept the suggested configuration and follow the instructions below on how to configure it for your Astro app.
+:::
+
+
+## Project Configuration
+
+### Static site
+
+If you build a simple Astro site with no adapters, you don’t need any extra configuration to your Astro project. 
+
+Edit the `Spacefile` file generated by the Space CLI on the root of your project. You will need to change the engine to `static`, add the `build` command, and serve the `dist` directory generated by Astro. 
```

Comment:
```suggestion
Make the following changes to the `Spacefile` file at the root of your project generated by the Space CLI. 

1. Change the engine to `static`.
2. Add the `build` command.
3. Serve the `dist` directory generated by Astro. 
```
So, to avoid calling more than half of Astro sites "simple" ... :sweat_smile: 

If no configuration to the Astro project itself is needed, let's just not say it! They won't even know it's missing!

Also, since we're at *three* changes to this file to make, it might make sense to list them out vs one long sentence. Also, I'll let @Jutanium weigh in here, because these instructions tell you what to do (e.g. serve the `dist` directory) in broad terms, but they really need the code example included to get it done. 

Right now, I'm thinking it's OK that the instructions read like general descriptions of what you're doing instead of telling you the exact code to write, with the example showing how to write the code that accomplishes it right there below. But, this is something Dan and I do think about carefully and I could be convinced by him that even the instructions should be more explicit.

Reply:
Right, totally agreed here. 

But I do think the combination of general instructions + the code snippet below works in terms with what you described here. 

The `change` in step one comes from the initially generated `Spacefile` (see `The Space CLI will try to detect the right configuration for your app.`), which might or might not be good, I didn't think we should mention it. Do you think we need a richer example or code snippet here?

Reply:
I am 100% for the list, good point 🤘🏼

Reply:
Agreed that the list does not have to contain everything you need to make the changes - the code sample does the work.
That said, I think we should match it with the terminology used in the code sample:

```suggestion
Make the following changes to the `Spacefile` file at the root of your project generated by the Space CLI. 
	
	1. Change the engine to `static`.
	2. Add Astro's build command to the list of `commands`.
	3. Serve the `dist` directory generated by Astro. 
```

Reply:
Approve, Dan! Let's ship this version!